src/core/CoreActionController.cpp

```diff
@@ -244,14 +232,13 @@ void CoreActionController::setStripPan( int nStrip, float fPanValue, bool bSelec
 	InstrumentList *pInstrList = pSong->getInstrumentList();
 
 	auto pInstr = pInstrList->get( nStrip );
-	pInstr->set_pan_l( fPan_L );
-	pInstr->set_pan_r( fPan_R );
+	pInstr->setPanWithRangeFrom0To1( fValue );
```

Comment:
Is there a benefit in keeping the range of the pan in the Mixerline from 0 to 1? How about we make it [-1,1] instead and note it done in the CHANGELOG? Since the values in existing songs will be properly mapped to the new values and the rotary is still a centered one, old users won't probably notice the difference while for new one the symmetric behavior might be more intuitive.

Reply:
I think it is ok to adopt [-1,1].